src/main/java/appeng/me/cache/GridStorageCache.java

```diff
@@ -155,28 +163,33 @@ public void populateGridStorage( final IGridStorage storage )
 
 	}
 
+	public <T extends IAEStack<T>, C extends IStorageChannel<T>> IMEInventoryHandler<T> getInventoryHandler( IStorageChannel<T> channel )
+	{
+		return (IMEInventoryHandler<T>) this.storageNetworks.computeIfAbsent( channel, c -> this.buildNetworkStorage( c ) );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1612)